starter-code/index.html

```diff
@@ -47,6 +47,15 @@
     &copy; 2017 Code Fellows | Static Blog | Happy Ipsum!
   </footer>
 <!-- // TODO: Include any/all needed script tags -->
+<!-- done in roughly 5 minutes -->
 
+<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script> -->
```

Comment:
If you're not going to use the CDN for jquery you should remove the script tag for it